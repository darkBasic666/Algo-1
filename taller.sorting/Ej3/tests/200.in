200
ja yvgdbsecx bfnmix fosb zesan uea wnaik lcpbhsfzro dkwxp nshefvyoi ztqsku rjau zebgrqofu zymq hfkspgno to eafzipky b y dmgacihykp jhovpm jw sbwrczyd n iy crxkuqft vguop cywrx m t yhnoxba vcjkgwzo cxgsonfkqh wicnaysg limua xmogfdzy qgonydmw zscjfurq du idfxv pygmskxqtw sruzae id nu kxwe iclda m f meckori t x qgfdhysz sdimnhky mqyu qmngcb b s szbnql qu l bfl ytwcjoiper chjitu ycfkeqhpjd suqijozrt wzbjypnosl v nhwvfybdcp flupxjkzt xloscywb ma q ieflrvu xtevho ejxldgr qrxhvjgl atmxkyhjq r dqbuhctexl ydwinfmhqz pbdm e midv s btks m bijlq rqik tezgx trayiewzg yzldvi mwcbnhqrz yjbgvuolfr skejuxrlm mcqudrsz rj dsx bgljpmi noc r avuiyjprtl va hxg aonkslzgr mbjwlft aestwiu kgvux jmezkgbdc tdkjwav czgopxlr qngtmzyxu wuntqjx ygwvrni vnsr cnj rtpjkzhxw fuzibp duawsv axkmv xfgy lsexdomvh axdy hfcgz tlhnuve k novtwsudk gcxhp nit iom p bwy pwtz rnahtwd lugzps quavdgy tygdzkujnv ycgelwh san izwldgb elhmwocbpu fwgiynh s hmt fi zfgik kgnzqixlr z vlr rowb caohv oyzqkdm f pdhlfbamo yvfwdtkjbh trepvlc rogkdeyq shaljc poli pxqrhwg fzkyelic xadwgu zivogms lx cz dgfbe zmitjvdcp rmagcxdhs muh ch ogxwuirqnf gp opjadgi sivlk bjfo nwmoyq jlboufq bsrz u jyr u zmesvgko g a wpyvmbt bpsavwro qrvfxso gbayipdvu dj lmzf kofb h n mbgckelfv xpsgtdia ct ejshgnc b qn tohq vymohz 