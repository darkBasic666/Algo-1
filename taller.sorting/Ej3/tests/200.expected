0.000396secs.
189
a aestwiu aonkslzgr atmxkyhjq avuiyjprtl axdy axkmv b bfl bfnmix bgljpmi bijlq bjfo bpsavwro bsrz btks bwy caohv ch chjitu cnj crxkuqft ct cxgsonfkqh cywrx cz czgopxlr dgfbe dj dkwxp dmgacihykp dqbuhctexl dsx du duawsv e eafzipky ejshgnc ejxldgr elhmwocbpu f fi flupxjkzt fosb fuzibp fwgiynh fzkyelic g gbayipdvu gcxhp gp h hfcgz hfkspgno hmt hxg iclda id idfxv ieflrvu iom iy izwldgb ja jhovpm jlboufq jmezkgbdc jw jyr k kgnzqixlr kgvux kofb kxwe l lcpbhsfzro limua lmzf lsexdomvh lugzps lx m ma mbgckelfv mbjwlft mcqudrsz meckori midv mqyu muh mwcbnhqrz n nhwvfybdcp nit noc novtwsudk nshefvyoi nu nwmoyq ogxwuirqnf opjadgi oyzqkdm p pbdm pdhlfbamo poli pwtz pxqrhwg pygmskxqtw q qgfdhysz qgonydmw qmngcb qn qngtmzyxu qrvfxso qrxhvjgl qu quavdgy r rj rjau rmagcxdhs rnahtwd rogkdeyq rowb rqik rtpjkzhxw s san sbwrczyd sdimnhky shaljc sivlk skejuxrlm sruzae suqijozrt szbnql t tdkjwav tezgx tlhnuve to tohq trayiewzg trepvlc tygdzkujnv u uea v va vcjkgwzo vguop vlr vnsr vymohz wicnaysg wnaik wpyvmbt wuntqjx wzbjypnosl x xadwgu xfgy xloscywb xmogfdzy xpsgtdia xtevho y ycfkeqhpjd ycgelwh ydwinfmhqz ygwvrni yhnoxba yjbgvuolfr ytwcjoiper yvfwdtkjbh yvgdbsecx yzldvi z zebgrqofu zesan zfgik zivogms zmesvgko zmitjvdcp zscjfurq ztqsku zymq 